### PR TITLE
Solve bug and enhancements

### PR DESCRIPTION
**Pull request by maplerichie**
Forget to inform for assignment before this PR. Thanks

### CODE REVIEW

**Comment by Penguin-12** [comment]
Could you also attach screenshots of the visual changes made? (like the new chat appbar, toast, etc)

File: lib/chat/view/ChatScreen.dart [hunk 1]
```diff
@@ -48,20 +46,23 @@ class ChatScreen extends StatelessWidget {
...
-              curve: Curves.easeOut);
+          _scrollController.animateTo(_scrollController.position.maxScrollExtent, duration: const Duration(milliseconds: 250), curve: Curves.easeOut);
         });
 
         return Scaffold(
-          appBar: ChatAppBar(
-            title: _role.requestTitle,
-            description: _role.requestDescription,
+          appBar: AppBar(
+            title: Text('Chat'),
```
Comment (Penguin-12) [inline lib/chat/view/ChatScreen.dart:54]: Any reason for swapping the custom appbar? Also description field is missing in the new one.

**Comment by maplerichie** [comment]
[Screenshot refrences](https://imgur.com/a/0oGisgA)

Thanks

**Comment by Penguin-12** [comment]
> [Screenshot refrences](https://imgur.com/a/0oGisgA)
> 
> Thanks

Nice work. Since the app wide theme is blue and white for now, please switch the logout confirmation dialog's yellow with blue.
Also the appbar and card is taking too much space on the chatscreen, hence please use the ChatAppBar widget instead to display the title and description of the request.

**Comment by maplerichie** [comment]
@Penguin-12 
Is there any character limit on request title and description?
The point of consideration of using Card is because once both title and description exceed some length, it will become ellipsis and unreadable. Thus acceptor will be lost of idea.
Or show it as the first chat bubble?

**Comment by Penguin-12** [comment]
> @Penguin-12
> Is there any character limit on request title and description?
> The point of consideration of using Card is because once both title and description exceed some length, it will become ellipsis and unreadable. Thus acceptor will be lost of idea.
> Or show it as the first chat bubble?

You can add a line limit to the app bar text and use ellipsis to account for overflow. We can always add an enlarged view to show the full title and description when the user taps on the appbar in some future issue.

**Comment by maplerichie** [comment]
[SliverAppBar implementation](https://imgur.com/a/MzzeYpk)

Commited to another branch

**Comment by maplerichie** [comment]
[Dialog implementation](https://imgur.com/a/SYBXOTK)

**Review by Penguin-12 — APPROVED** [review]

**Comment by Penguin-12** [comment]
Well done. Congratulations on your first contribution to the repository. For the future, please wait for assignment of issue before begining, also please solve one issue at a time as it is easier to maintain and enables us to keep clean records.

**Comment by Penguin-12** [comment]
Closes #21, #22, #26, #35

**Comment by maplerichie** [comment]
> Well done. Congratulations on your first contribution to the repository. For the future, please wait for assignment of issue before begining, also please solve one issue at a time as it is easier to maintain and enables us to keep clean records.

Ya, missed out the steps of conduct. 
Thanks

**Pull request merged**